packages/@react-spectrum/tooltip/src/TooltipTrigger.tsx

```diff
@@ -45,20 +45,20 @@ function TooltipTrigger(props: SpectrumTooltipTriggerProps) {
 
   let [borderRadius, setBorderRadius] = useState(0);
   useLayoutEffect(() => {
-    if (overlayRef.current) {
+    if (overlayRef.current && state.isOpen) {
       let spectrumBorderRadius = window.getComputedStyle(overlayRef.current).borderRadius;
       if (spectrumBorderRadius !== '') {
         setBorderRadius(parseInt(spectrumBorderRadius, 10));
       }
     }
-  }, [overlayRef]);
+  }, [state.isOpen]);
```

Comment:
this should still have overlayRef, along with setBorderRadius

Reply:
I can add overlayRef, but isn't setBorderRadius guaranteed to be stable?

Reply:
it is, i forget if the eslint thing can figure that out?